tests/Unit/Controller/Web/OrganizationWebControllerTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Unit\Controller\Web;
+
+use App\Controller\Web\OrganizationWebController;
+use App\DataFixtures\Entity\OrganizationFixtures;
+use App\Tests\AbstractWebTestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Uid\Uuid;
+
+class OrganizationWebControllerTest extends AbstractWebTestCase
+{
+    public function testListRouteRendersHTMLSuccessfully(): void
```

Comment:
```suggestion
{
    protected function setUp(): void
    {
        $this->client = self::createClient();
    }
    
    public function testListRouteRendersHTMLSuccessfully(): void
```

Considerando que todos os testes utilizam `$client = static::createClient();`, seria recomendável adicioná-lo à função `setup()` para garantimos que o cliente seja criado antes de cada teste ser executado, eliminando a necessidade de repetir essa linha em cada método de teste individualmente. 

observe como é feito aqui : 
https://github.com/ecossistema-aurora/aurora/blob/c4b7add6dd08c24e27cd9eac616fbff3a4e3de25/tests/Unit/Controller/Web/SpaceWebControllerTest.php#L14-L26